src/tests/ftest/server/daos_server_config.yaml

```diff
@@ -93,10 +93,20 @@ server_config_val: !mux
       - "nr_hugepages"
       - 1000000000000000000
       - "FAIL"
-  nr_hugepages_negative_int:
+  nr_hugepages_large_negative_int:
     config_val:
       - "nr_hugepages"
       - -100
+      - "FAIL"
+  nr_hugepages_small_negative_int:
```

Comment:
maybe this is a question for a test developer @phender - would the name of this case better be "hugepages_disabled"?

Also with this test are there bdevs in the configuration or not?

Reply:
I think that would be a better name, however I'm going to remove this added test case for the following reason.

The result of disabling hugepages is not deterministic in this case, although the validation phase during daos_server can be determined and would indeed pass for `nr_hugepages: -1`, subsequent calls to commands that use the SPDK bindings may fail depending on system environment (the daos_server_config test performs storage format and scan). 
Some examples:
- If the system has NVMe SSDs currently bound to VFIO but no hugepages have been allocated then we will see permission denied errors from storage scan.
- If the system has NVMe SSDs currently bound to VFIO by a different user and hugepages have been allocated then we will probably see VFIO device in use errors from storage scan.
- If SSDs are bound to the kernel driver then storage scan will complete successfully.

Disabling hugepages is a documented edge case and as such I believe we have sufficient test coverage in the control plane unit tests that verify the setting behaviour.

Reply:
bdevs are not configured in this test